docroot/sites/settings/cgov_core.settings.php

```diff
@@ -61,15 +61,16 @@
 
 // on enironments that have SAML, route to Okta upon logout
 switch ($env) {
-  case '01live':
-    $config['simplesamlphp_auth.settings']['logout_goto_url'] = 'https://iam.cancer.gov/login/signout';
   case '01dev':
   case 'dev':
   case '01test':
   case 'test':
   case 'int':
     $config['simplesamlphp_auth.settings']['logout_goto_url'] = 'https://iam-stage.cancer.gov/login/signout';
+    break;
+  // Production for default, including 01live
```

Comment:
Can you please move the comment right above `$config`? It breaks up the switch readability.